easybuild/easyconfigs/c/CLISP/patch-modules_readline_readline.lisp

```diff
@@ -0,0 +1,15 @@
+$NetBSD: patch-modules_readline_readline.lisp,v 1.1 2016/09/20 14:10:25 wiz Exp $
```

Comment:
@JackPerdue where did you get this patch from exactly, do you have an URL for it?